cmd/listserving/list-collections.go

```diff
@@ -9,7 +9,7 @@ import (
 	"strings"
 
 	"github.com/foursquare/fsgo/net/thriftrpc"
-	"github.com/foursquare/quiver/gen"
+	"quiver/internal/gen"
```

Comment:
I think you need to remove all the internal changes now that you reverted that bit of your original patch